infrastructure/iac/template/styles.css

```diff
@@ -20,7 +21,7 @@
 }
 
 ::-webkit-scrollbar-thumb:hover {
-  background: var(--scroll-bar-hover-background-color);
+  background: var();
```

Comment:
don't we need the var --scroll-bar-hover-background-color here

Reply:
We don't replace it anywhere in IntelliJ or VSCode I'm not sure it's actually used, maybe I'm missing something? Or maybe we use it in VS or Eclipse ?